src/gamma.jl

```diff
@@ -317,6 +317,15 @@ function zeta(s::ComplexOrReal{Float64}, z::ComplexOrReal{Float64})
     return ζ
 end
 
+
+"""
+    hurwitz_zeta(s, z)
+
+    An alias for zeta(s, z)
+"""
+hurwitz_zeta(s::ComplexOrReal{Float64}, z::ComplexOrReal{Float64}) = zeta(s, z)
```

Comment:
I guess this is an issue with how Julia does docs, but in my initial reading I hadn't realised that Julia had a Hurwitz zeta function already. I know the information is there, but it wasn't obvious  for a newbie (like me). If its not standard for Julia to include aliases like this let me know, but it seemed like a trivial way to make life easier for users? 

I did the same with Li=polylog. I realised after writing code that the naming convention for Julia would usually give us something like polylog?